src/walletconnect/index.js

```diff
@@ -2,20 +2,21 @@
 /* eslint-disable no-fallthrough */
 import { BigNumber } from "bignumber.js";
 import eip55 from "eip55";
+import InputDataDecoder from "ethereum-input-data-decoder";
+import sha from "sha.js";
+import { bufferToHex } from "ethereumjs-util";
 import { getAccountBridge } from "../bridge";
 import { getCryptoCurrencyById } from "../currencies";
 import type { Account, Transaction } from "../types";
 import type {
   TypedMessageData,
   TypedMessage,
 } from "../families/ethereum/types";
-import {
-  stringHash,
-  domainHash,
-  messageHash,
-} from "../families/ethereum/hw-signMessage";
+import { domainHash, messageHash } from "../families/ethereum/hw-signMessage";
 import type { MessageData } from "../hw/signMessage/types";
 
+const decoder = new InputDataDecoder(`${__dirname}/erc20.abi.json`);
```

Comment:
I'm pretty sure will not work in React native.
I'm afraid we need to make the erc20 abi a .js file that we import